irispy/new_sji.py

```diff
@@ -270,13 +273,13 @@ def __repr__(self):
            inst_start=instance_start,
            inst_end=instance_end,
            seq_shape=self.dimensions,
-           axis_types=self.world_axis_physical_types)
+           axis_types=self.world_axis_physical_coords)
 
     def __getitem__(self, item):
-        return utils.sequence.slice_sequence(self, item)
+        return self.index_as_cube[item]
 
     @property
-    def cube_dimensions(self):
+    def dimensions(self):
         return self.cube_like_dimensions
 
     @property
```

Comment:
Sorry, I think I told you to call this property ```world_axis_physical_coords```.  It was a typo.  It should have the same name as the ```NDCubeSequence``` method which is ```world_axis_physical_types```.  My mistake.  THis means it also needs changing in line 276 well.

Reply:
Modified